com.unity.multiplayer.mlapi/Runtime/Core/RpcBatcher.cs

```diff
@@ -10,29 +10,30 @@
 
 namespace MLAPI
 {
-    class MessageBatcher
+    internal class RpcBatcher
     {
         public class SendStream
         {
-            public byte channel;
-            public PooledBitStream Stream = PooledBitStream.Get();
+            public byte Channel;
+            public PooledBitStream Stream;
             public PooledBitWriter Writer;
-            public bool Empty = true;
+            public bool IsEmpty = true;
 
             public SendStream()
             {
+                Stream = PooledBitStream.Get();
                 Writer = PooledBitWriter.Get(Stream);
             }
         }
 
         // Stores the stream of batched RPC to send to each client, by ClientId
-        private Dictionary<ulong, SendStream> SendDict = new Dictionary<ulong, SendStream>();
+        private readonly Dictionary<ulong, SendStream> SendDict = new Dictionary<ulong, SendStream>();
 
         // Used to store targets, internally
         private ulong[] TargetList = new ulong[0];
 
         // Used to mark longer lengths. Works because we can't have zero-sized messages
-        private const byte LongLenMarker = 0;
+        private const byte LongLengthMarker = 0;
```

Comment:
Why? `len` is pretty clear, no? Does that mean `pos`, `rot` and `vec` would also be too short? Is this covered in the standard coding convention?

Reply:
oh, `Len` is alright but I think it should at least be `k_LongLenMarker`. I will make the change!